.github/prompts/code-review-local-changes.prompt.md

```diff
@@ -0,0 +1,306 @@
+﻿---

```

Comment:
The code review prompt file not only evaluates the areas where changes have occurred, but also assesses the parts that haven't changed. Is this the expected behavior?